students/JamesTraub/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+# Lesson 3 - Series 1
+
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print("Create and display a list of fruit.")
+print(fruit)
+print()
+
+inp_fruit = input('Please enter a name of a fruit to add to the end of list. ')
+fruit.append(inp_fruit)
+print(fruit)
+print()
+
+fruit_number = int(input('Please enter the number that correspends with one fruit in the list. '))
+for i, nfruit in enumerate(fruit):
+    if fruit_number == (i + 1):
+        print(fruit_number, nfruit)
```

Comment:
instead of iterating over list, use index access to get fruit by number:
`fruit[fruit_number + 1]`